### ISSUE TITLE
Exploring Semantic Spaces - Hamilton, Leskovec & Jurafsky 2016

### ISSUE THREAD

**Issue by jamesallenevans** [Computational-Content-Analysis-2020/Readings-Responses#34]
Hamilton, William H., Jure Leskovec, Dan Jurafsky. 2016. [“Diachronic Word Embeddings Reveal Statistical Laws of Semantic Change.”](https://www.aclweb.org/anthology/P16-1141/) arXiv preprint arXiv:1605.09096.

**Comment by sunying2018** [comment]
This paper is very innovative to implement word embeddings to understand how words change diachronically. It proposes two quantitative laws of semantic change to answer the two important questions that what is the role of frequency in semantic change and what is the relationship between semantic change and polysemy.  Three embedding algorithms are implemented in this paper.  I have a question for the evaluation these three algorithms in the task of discovering shifts from data. Since this evaluation is only based on ENGFIC data which is dominated by scientific terms, and mainly evaluated their performance on the top-10 words discovered by each method,  I am wondering whether the natural and characteristic of this corpora and the evaluation criteria (top-k) will affect the results in this article?

**Comment by jsmono** [comment]
The authors proposed an interesting way to analyze the shift of word meanings but I am a bit confused about why they selected the six datasets. The authors briefly explained their choice of COHA and Google books but was not sufficient to explain the benefits of them. For example, why they selected three English datasets? Why did they choose French German and Chinese to test their method? What are some of the biases of the datasets? Without a better understanding of their choice, it was a biit challenging for me to understand why EngAll performed better than COHA and believe that the size of the corpus was the only reason for the lack of performance of COHA.

**Comment by chun-hu** [comment]
I agree with @jsmono that the lack of explanation on the choice of datasets gives us less information on how we should evaluate these methods. I'm also wondering how we should explain the variations in the magnitudes of linear effect on rates of semantic change. What kind of factors leads to these variations?

**Comment by cindychu** [comment]
In this paper, the author mentioned three different techniques for word embedding, PPMI, SVD and SGNS and I am wondering what are the major differences of these techniques, and what are their strength and relative weakness? 

Also this study aimed to conduct diachronic analysis for linguistic features, however, for diachronic analysis, the sample they have across time could be greatly intrinsically biased considering the way we store and retrieve resources have changed dramatically across time. How could the authors properly fix the biased sample across time  problem?

**Comment by adarshmathew** [comment]
My question is on the construction of the common vocabulary that makes comparisons across time-frames feasible. In the code provided, only the common set of words across models are used. 

1. Doesn't this result in a very small vocabulary if you have a large time series (narrowing common vocabulary with increase in time periods)?
2. Doesn't this also exclude substitution effects -- while 'gay' moved from meaning 'happy' to 'homosexual', some other word could've arisen to become the go-to word for happy, and the model may not be able to identify this new word.
3. Wouldn't this method also exclude new words in later periods, as they wouldn't be present in the first few periods? How do we work around this?

**Comment by ckoerner648** [comment]
In the detection of know semantic shifts, PPMI “was noticeably worse” (p. 5) than SVD. Why did SVD perform so much better than PPMI if SVD is just a low dimensional approximation of PPMI? The authors only suggest that “dimensionality reduction acts as a form of regularization” (p. 3), but I wonder how this works and why the difference in performance is so remarkable.
